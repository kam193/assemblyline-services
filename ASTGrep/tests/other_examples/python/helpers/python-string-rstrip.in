b"aaa   ".rstrip()
"vvvsss".rstrip("s")

not_a_string.rstrip()