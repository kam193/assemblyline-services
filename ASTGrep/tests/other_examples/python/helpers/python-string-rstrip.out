b'aaa'
'vvv'

not_a_string.rstrip()